frontend/src/components/plugins/forms/AnalyzerConfigForm.jsx

```diff
@@ -0,0 +1,612 @@
+            <FormGroup>
+              <Row>
+                <Col md={2}>
+                  <Label className="me-2 mb-0 required" for="analyzer-url">
+                    Url:
+                  </Label>
+                </Col>
+                <Col>
+                  <Input
+                    id="analyzer-url"
+                    type="text"
+                    name="url"
+                    value={formik.values.url}
+                    onBlur={formik.handleBlur}
+                    onChange={formik.handleChange}
+                    valid={!formik.errors.url && formik.touched.url}
+                    invalid={formik.errors.url && formik.touched.url}
+                    className="bg-darker border-0"
+                    placeholder="ex: http://www.service.com/"
+                  />
+                  <div className="d-flex flex-column mt-1">
+                    <small className="mt-0 fst-italic">
+                      URL of the instance you want to connect to
+                    </small>
+                    {formik.touched.url && formik.errors.url && (
+                      <small className="text-danger">{formik.errors.url}</small>
+                    )}
+                  </div>
+                </Col>
+              </Row>
+            </FormGroup>
+            <FormGroup>
+              <Row>
+                <Col md={2}>
+                  <Label className="me-2 mb-0 required" for="http_method">
+                    HTTP method:
+                  </Label>
+                </Col>
+                <Col>
+                  {Object.values(HTTPMethods).map((method) => (
+                    <FormGroup check inline key={`http_method__${method}`}>
+                      <Input
+                        id={`http_method__${method}`}
+                        type="radio"
+                        name="http_method"
+                        value={method}
+                        checked={formik.values.http_method.includes(method)}
+                        onBlur={formik.handleBlur}
+                        onChange={formik.handleChange}
+                      />
+                      <Label check for={`http_method__${method}`}>
+                        {method.toUpperCase()}
+                      </Label>
+                    </FormGroup>
+                  ))}
+                </Col>
+              </Row>
+              {formik.values.http_method === HTTPMethods.GET ? (
+                <Row className="mt-2">
+                  <Col md={10} className="offset-2">
+                    <div className="d-flex flex-column">
+                      <small className="fst-italic">
+                        <strong className="fst-italic">Request formats</strong>
+                        <br />
+                        Query string (default):&nbsp;
+                        <strong className="text-primary fst-italic">
+                          http://www.service.com?&lt;param_name&gt;=&lt;observable&gt;
```

Comment:
remove the < and > from the param_name, in this way is clearer that it's not populated automatically likes <observable> and <api_key>